pdr_backend/lake/test/test_gql_data_factory.py

```diff
@@ -352,3 +351,312 @@ def test_load_missing_parquet(
+    ),
+    (
+        "0xaaaa-1699038000-0xd2a24cb4ff2584bad80ff5f109034a891c3d88dd",
+        "0xaaaa",  # contract
+        1699038000,  # Nov 03 2023 19:00:00 GMT
+        "0xd2a24cb4ff2584bad80ff5f109034a891c3d88dd",  # user
+        "3.4600000000000004",  # stake
+        "7.041434095860760067",  # payout
+        False,  # predictedValue
+        False,  # trueValue
+        1699038002,  # timestamp
+    ),
+    (
+        "0xbnnn-1699124400-0xd2a24cb4ff2584bad80ff5f109034a891c3d88dd",
+        "0xbnnn",  # contract
+        1699124400,  # Nov 04 2023 19:00:00 GMT
+        "0xd2a24cb4ff2584bad80ff5f109034a891c3d88dd",  # user
+        "3.4600000000000004",  # stake
+        "7.160056238874628619",  # payout
+        True,  # predictedValue
+        True,  # trueValue
+        1699124402,  # timestamp
+    ),
+    (
+        "0xeeee-1699300800-0xd2a24cb4ff2584bad80ff5f109034a891c3d88dd",
+        "0xeeee",  # contract
+        1699300800,  # Nov 06 2023 19:00:00 GMT
+        "0xd2a24cb4ff2584bad80ff5f109034a891c3d88dd",  # user
+        "3.4600000000000004",  # stake
+        "0",  # payout
+        True,  # predictedValue
+        False,  # trueValue
+        1699300802,  # timestamp
+    ),
+]
+
+
+def mock_payouts():
+    payouts = []
+    for _payout in _PAYOUTS:
+        (
+            ID,
+            contract,
+            slot,
+            user,
+            stake,
+            payout,
+            predvalue,
+            truevalue,
+            timestamp,
+        ) = _payout
+
+        payouts.append(
+            {
+                "ID": ID,
+                "contract": contract,
+                "slot": slot,
+                "user": user,
+                "stake": stake,
+                "payout": payout,
+                "predvalue": predvalue,
+                "truevalue": truevalue,
+                "timestamp": timestamp,
+            }
+        )
+    return payouts
+
+
+@enforce_types
+def test_post_tables(tmpdir):
+    # setup test
+    st_timestr = "2023-11-02_0:00"
+    fin_timestr = "2023-11-07_0:00"
+
+    _, gql_data_factory = _gql_data_factory(
+        tmpdir,
+        "binanceus ETH/USDT h 5m",
+        st_timestr,
+        fin_timestr,
+    )
+
+    predictions = mock_predictions()
+    truevals = mock_truevals()
+    payouts = mock_payouts()
+
+    gql_dfs = {
+        "pdr_predictions": pl.DataFrame(predictions),
+        "pdr_truevals": pl.DataFrame(truevals),
+        "pdr_payouts": pl.DataFrame(payouts),
+    }
+
+    print("\n\ngql_dfs", gql_dfs)
+
+    # process post truevals
+    gql_data_factory._post_fetch_process_truevals(gql_dfs)
+    predictions_df = pl.read_parquet("pdr_predictions.parquet")
+    print(">>> truevals saved_parquet predictions_df", predictions_df)
+
+    # process post payouts
+    gql_data_factory._post_fetch_process_payouts(gql_dfs)
+    predictions_df = pl.read_parquet("pdr_predictions.parquet")
+    print(">>> payouts saved_parquet predictions_df", predictions_df)
```

Comment:
verify that after the post_processing, show/assert the new, updated records, as you'd expect them